bots/psuteam7bot/movement.js

```diff
@@ -0,0 +1,408 @@
+const movement = {}
+
+//Array for getting direction after rotation
+movement.directions = [{ x: 0, y: -1 }, { x: 1, y: -1 }, { x: -1, y: -1 }, { x: 1, y: 0 }, { x: -1, y: 0 }, { x: 0, y: 1 }, { x: 1, y: 1 }, { x: -1, y: 1 }]
+
+/*Return relative position of point A from point B
+*Input: A - a 'position/ location' object {x, y}
+*       B - a 'position/ location' object {x, y}
+*Output:    retVal - an object {x, y}, containing how many x and y steps needed for A to reach B
+*/
+movement.getRelativePosition = (A, B) => {
```

Comment:
FYI the current structure will find the position of A from B. Since I imagine that what is being passed in will be `(position, destination)`, this will find the relative position from destination, which is probably not what we want if we are trying to get to destination from position